src/components/structures/MatrixChat.tsx

```diff
@@ -2085,3 +2085,10 @@ export default class MatrixChat extends React.PureComponent<IProps, IState> {
         </ErrorBoundary>;
     }
 }
+
+export function isLoggedIn(): boolean {
+    // JRS: Maybe we should move the step that writes this to the window out of
+    // `element-web` and into this file?
```

Comment:
ew :/ or probably shift the data of whether the app's in the 'logged in' state into a store somewhere.

Reply:
I filed https://github.com/vector-im/element-web/issues/15034 for this and added a comment here.